py/bitbox02/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -122,6 +122,8 @@ class AttestationPubkeyInfo(TypedDict):
 
 MIN_BITBOX02_MULTI_FIRMWARE_VERSION = semver.VersionInfo(6, 0, 0)
 MIN_BITBOX02_BTCONLY_FIRMWARE_VERSION = semver.VersionInfo(6, 0, 0)
```

Comment:
Maybe rename this to `MIN_SUPPORTED_*` for consistency?

Reply:
Done